dart-github-dart-lang/.travis.yml

```diff
@@ -1,14 +1,24 @@
 language: dart
 
-script: ./tool/travis.sh
-
 # Speed up builds by using containerization. Disable this if you need to use
 # sudo in your scripts.
 sudo: false
 
+dart:
+ - dev
+ - stable
+
+dart_task:
+ - test: --platform dartium
```

Comment:
I suspect most will be just vm. My 0.02 would be to comment out the dartium and firefox tasks, w/ a comment letting people to know to re-enable if appropriate for their projects.